network-api/networkapi/buyersguide/urls.py

```diff
@@ -5,6 +5,7 @@
 urlpatterns = [
     url(r'^$', views.buyersguide_home, name='buyersguide-home'),
     url(r'^about/', views.about_view, name='about-view'),
+    url(r'^category/(?P<categoryname>[\w\W]+)/', views.category_view, name='category-view'),
```

Comment:
Might want to use `categories/` for consistency with the recently renamed `products/` route fragment. @cadecairos may have thoughts.

Reply:
yes, lets keep the resource names in our routes pluralized

Reply:
:+1: